content/docs/details/scaffolder/index.md

```diff
@@ -0,0 +1,385 @@
+---
+title: Scaffolder
+publishedDate: '2022-05-13T10:53:00.0Z'
+description: Initialize projects and trigger automation from within Backstage. You can prompt parameter inputs and combine scaffolder actions in order to build a scaffolder template to run within Backstage.
+---
+
+# Overview
+Imagine you’re an engineer looking to create a new microservice. You want to get started as quickly as possible, with minimal boilerplate and red-tape to jump through. At the same time, engineering organizations benefit from having consistency in production, and often put gates in place to enforce it.
+
+Instead of creating blockers for engineering teams, you can use the Roadie scaffolder and quickly create new microservices, while helping to ensure that production remains mostly consistent.
+
+Engineers can choose a pre-defined software template, fill out a few form fields to provide values like the name of the GitHub repo that the new service will occupy, and click a button to run the template and create a new service.
+
+# Components of a Template
+You can configure a Scaffolder _template_ to be triggered on demand by Roadie users. Your template contains one or more `steps` which run sequentially during execution of a template. You may optionally specify all `steps` while the template execution is running inside an ephemeral container that is destroyed after the execution completes.
```

Comment:
I'd move this paragraph down a bit. You gotta explain what a template is before you say what the steps are.